packages/create-vite/src/index.ts

```diff
@@ -143,6 +143,12 @@ const FRAMEWORKS: Framework[] = [
         color: cyan,
         customCommand: 'npm create react-router@latest TARGET_DIR',
       },
+      {
+        name: 'custom-tanstack-router',
+        display: 'TanStack Router ↗',
+        color: cyan,
+        customCommand: 'npm create -- create-tsrouter-app@latest TARGET_DIR --framework react',
```

Comment:
shouldn't the `create-` be removed here?
```suggestion
        customCommand: 'npm create tsrouter-app@latest -- TARGET_DIR --framework react',
```

Reply:
Oh, let me try that!